<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>653</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>601</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Get All</string>
     </attribute>
     <widget class="QTableWidget" name="allEntityTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>901</width>
        <height>541</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>112</number>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Full name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sex</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date of birth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Money</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Is in debt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Debt</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="getAllButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get all</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Get One</string>
     </attribute>
     <widget class="QPushButton" name="getOneButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>901</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="oneEntityTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>901</width>
        <height>71</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>112</number>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Full name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sex</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date of birth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Money</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Is in debt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Debt</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="idInput_One">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>251</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorLable_One">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>340</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>891</width>
        <height>561</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_firstName">
         <property name="text">
          <string>First name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_firstName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_middleName">
         <property name="text">
          <string>Middle name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_middleName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_lastName">
         <property name="text">
          <string>Last name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_lastName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_sex">
         <property name="text">
          <string>Sex</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="input_sex">
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dateOfBirth">
         <property name="text">
          <string>Date of birth</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="input_dateOfBirth">
         <property name="minimumDateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>1900</year>
           <month>9</month>
           <day>14</day>
          </datetime>
         </property>
         <property name="maximumDate">
          <date>
           <year>2024</year>
           <month>12</month>
           <day>31</day>
          </date>
         </property>
         <property name="minimumDate">
          <date>
           <year>1900</year>
           <month>9</month>
           <day>14</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>M/d/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_address">
         <property name="text">
          <string>Address</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_address"/>
       </item>
       <item>
        <widget class="QLabel" name="label_amounOfM">
         <property name="text">
          <string>Amount of money</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="input_amountOfM">
         <property name="maximum">
          <double>9999999999999999538762658202121142272.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_isInDebt">
         <property name="text">
          <string>Is in debt?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="input_isInDebt">
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_amountOfD">
         <property name="text">
          <string>Amount of debt</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="input_amountOfD">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>9999999999999999538762658202121142272.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Delete</string>
     </attribute>
     <widget class="QLabel" name="ErrorLable_Delete">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>340</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>901</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="idInput_Delete">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>251</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="deleteEntityTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>901</width>
        <height>71</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>112</number>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Full name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sex</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date of birth</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Money</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Is in debt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Debt</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="question">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>380</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Are you sure?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="yesButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>420</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="noButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>420</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEntity">
    <property name="title">
     <string>Entity</string>
    </property>
    <addaction name="actionCustomer"/>
   </widget>
   <addaction name="menuEntity"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCustomer">
   <property name="text">
    <string>Customer</string>
   </property>
  </action>
  <action name="actiondeleteById">
   <property name="text">
    <string>deleteById</string>
   </property>
  </action>
  <action name="actiongetAll">
   <property name="text">
    <string>getAll</string>
   </property>
  </action>
  <action name="actiongetOne">
   <property name="text">
    <string>getOne</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
